validation/contextual-services/topsites-click.1.event.pass.json

```diff
@@ -0,0 +1,16 @@
+{
+  "context_id": "{94642acb-4996-034b-916c-147da723cc41}",
+  "advertiser": "amazon",
+  "reporting_url": "https://bridge.sfo1.ap01.net/ctp?version=16.0.0&key=1598991514900100001.1&ci=15 98991514792.12747&ctag=1598291839300100006&aespFlag=altinst&ip=%IP%&ua=%UA%",
```

Comment:
Are these real urls and if so do they reveal anything sensitive?  Would it be better to use placeholder values?

Reply:
Yes, they are. Though there are a few query parameters to be populated by the data processor before sharing with the partner.